Formula/astroid.rb

```diff
@@ -29,11 +29,13 @@ def install
       "-DDISABLE_PLUGINS:BOOL=OFF",
       "-DDISABLE_TERMINAL:BOOL=OFF",
       "-DENABLE_PROFILING:BOOL=OFF",
-      "-H.",
-      "-Bbuild",
-      "-GNinja",
+      "-GUnix\ Makefiles",
     ]
-    system "cmake", *args
+    args += [
+      "--release=git",
+      "-GNinja",
```

Comment:
Isn't two -G.. directives added now?

Reply:
yes but the hack is that the last one takes precedence

Reply:
Ok